### DISCUSSION TITLE
Write Access Request

### DISCUSSION THREAD

**Discussion by dbusse-wcr7** [netbox-community/devicetype-library#855] [General]
# Hi all,

I find myself visiting an contributing to this repository a *lot* a lately and wanted to know if any new moderators / people with write access for this repository were needed / wanted. If so, I would be happy to help.

**Comment by ryanmerolle** [comment]
We will have to decline at the moment as we don't have guidelines for maintainers of this repo outside of the current NetBox team.  We will publish a guide to becoming a maintainer for this project soon.

**Reply by dbusse-wcr7** [reply]
Thanks for the info!

**Reply by stereu** [reply]
Any updates on this right now?

**Reply by phillf** [reply]
Are there still plans to amend the CONTRIBUTING.md in regards to being an maintainer?

**Reply by danner26** [reply]
Hello, we do not need additional assistance at the moment, but if the need changes we will update the community. Thank you for the interest
